compiler/src/iree/compiler/Dialect/Flow/Transforms/Passes.cpp

```diff
@@ -432,6 +442,17 @@ void registerFlowTransformPassPipeline() {
           passManager.dump();
         });
       });
+
+  PassPipelineRegistration<> flowDispatchRegionCreationPipeline(
+      "iree-flow-dispatch-region-creation-pipeline",
+      "Flag used to run passes that form dispatch regions",
+      [](OpPassManager &passManager) {
+        addDispatchRegionCreationPasses(passManager);
+        LLVM_DEBUG({
```

Comment:
not sure about having prints like this - these are really messy and makes -debug hard to use - you can use print-ir-after to do this but here you don't even need it - if you're running this pipeline you'll get the IR after it completes anyway

Reply:
True, removed this and the one above